src/olympia/constants/base.py

```diff
@@ -415,12 +415,10 @@
 
 # Types of Canned Responses for reviewer tools.
 CANNED_RESPONSE_ADDON = 1
-CANNED_RESPONSE_APP = 2
 CANNED_RESPONSE_PERSONA = 3
 
 CANNED_RESPONSE_CHOICES = {
     CANNED_RESPONSE_ADDON: _('Add-on'),
-    CANNED_RESPONSE_APP: _('App'),
```

Comment:
`CANNED_RESPONSE_APP` isn't used directly, but `CANNED_RESPONSE_CHOICES` is used in the `CannedResponse` model.  Anyway, there are canned responses with type=3 still around (dev has them, for example) so if you're removing the last reference to what that id means then you should remove them with a migration.